src/api/translation/open_source/open_source_translate_api.py

```diff
@@ -1,22 +1,47 @@
 from typing import List, Dict
 import pandas as pd
+from langchain_core.documents import Document
+from langchain_text_splitters import RecursiveCharacterTextSplitter
 
 from src.api.translation.i_translate_api import ITranslateAPI
 from src.api.translation.translation_result import TranslationResult
 
 
+def _combine_translation_chunks(translations: List[str], lengths: List[int]):
```

Comment:
Why is this function separted from the open source class?

Reply:
This is a static function that doesn't rely on the open source class. It doesn't take in self.